daemon/tests/happy_path.rs

```diff
@@ -77,7 +77,7 @@ fn assert_eq_order(mut published: CfdOrder, received: CfdOrder) {
 
     // Hard-coded to match the dummy_new_order()
     assert_eq!(received.opening_fee.unwrap(), Amount::from_sat(2));
-    assert_eq!(received.funding_rate_hourly_percent, "0.001");
+    assert_eq!(received.funding_rate_hourly_percent, "0.00100");
```

Comment:
Note to self: Why do we need these two additional zeros here? We should not fix the percentage but fix it in the UI...